ansible/roles/omero-build/tasks/main.yml

```diff
@@ -35,6 +35,13 @@
     - python-pip
     - python-virtualenv
 
+- name: system packages | pip install packages
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - genshi
```

Comment:
Understood. We might want to split omero-build from bf-build eventually unless there is a reason to keep them together /cc @manics @kennethgillen